.github/workflows/full-qa.yml

```diff
@@ -0,0 +1,114 @@
+name: Full QA
+
+# all pull requests
+on: pull_request
+
+jobs:
+  unit-tests-p2:
+    name: Unit Tests / Polymer 2 on the CI agent
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node 12.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+
+      - name: Check out the source code
+        uses: actions/checkout@v2
+
+      - name: Install global npm dependencies
+        # bower is needed to run 'bower install'
+        # polymer-cli is needed to run the lint step
+        # web-component-tester is needed to run the test step
+        run: "npm install --quiet --no-progress --global bower polymer-cli web-component-tester"
+
+      - name: Install project npm dependencies
+        run: "npm install --quiet --no-progress"
+
+      - name: Install project Bower dependencies
+        run: "bower install --quiet"
+
+      - name: Run automated magi-cli checks
+        run: "npm run check"
+
+      - name: Run a linter
+        run: "npm run lint"
+
+      # the full set of environments is tested with Polymer 3 below
+      - name: Run unit tests locally (in the VM instance running this job)
+        run: "xvfb-run -s '-screen 0 1024x768x24' wct"
```

Comment:
This seems different from the Travis script, which runs `wct --env saucelabs` for both P2 and P3, for full runs.

Not sure if there are big differences between P2 and P3 to justify full runs though.

Reply:
Indeed, this is different from how it was on Travis, and for exact same reason you've mentioned. I assume that if the P3 version passes tests in all browsers on SauceLabs, then for the P2 version it would be enough to run all tests only in a couple of browsers.

The P3 version is automatically generated from P2 by a script. No changes are expected.